picsite/sharepic/templates/sharepic/index.html

```diff
@@ -0,0 +1,9 @@
+<html>
+<head>
```

Comment:
It's better to write "base.html"
If not, it's better to write <!DOCTYPE html> stuff